etc/firewalld/services/tendrl-monitoring-integration.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
We could store this file in `firewalld/tendrl-monitoring-integration.xml` to make it more visible instead of suggested `etc/firewalld/services/tendrl-monitoring-integration.xml`.

Reply:
Done